docs/sources/articles/networking.md

```diff
@@ -12,7 +12,9 @@ private range defined by [RFC 1918](http://tools.ietf.org/html/rfc1918)
 that are not in use on the host machine, and assigns it to `docker0`.
 Docker made the choice `172.17.42.1/16` when I started it a few minutes
 ago, for example — a 16-bit netmask providing 65,534 addresses for the
-host machine and its containers.
+host machine and its containers. Mac address is generated from ip to 
+avoid arp collisions and uses a range from 02:42:ac:11:00:00 to 
+02:42:ac:11:ff:ff.
```

Comment:
can you please change the wording a little?

perhaps something like

 ```
The MAC address is generated using the IP address allocated to the container to avoid ARP collisions, using a range from `02:42:ac:11:00:00` to `02:42:ac:11:ff:ff`.
```

mmm, and I'm betting I didn't get the capitalisation right

@fredlf @jamtur01